beacon-node-oapi.yaml

```diff
@@ -157,11 +156,11 @@ paths:
     $ref: "./apis/builder/states/expected_withdrawals.yaml"
 
   /eth/v2/debug/beacon/states/{state_id}:
-    $ref: './apis/debug/state.v2.yaml'
+    $ref: "./apis/debug/state.v2.yaml"
```

Comment:
the formatting changes in this PR got a bit out of hand, we should revert those changes before merging

Reply:
I'd suggest keeping these, lots of the formatting in these yaml files are inconsistent, there are single and double quotes everywhere, this PR just tries to move them into one consistent direction (double quote)

Reply:
If you want to tidy up the formatting perhaps break it out into a separate PR.  It would also make this PR far more readable.

Reply:
this PR initially had some formatting changes which was ok but now it has more formatting changes than anything else, we can easily merge a PR that fixes that separately and maybe even consider adding a linter to CI, if you feel strongly about this @0x00101010, just don't see why a PR that proposes PeerDAS changes should have all those changes in it

Reply:
Cool, I think you're right, I didn't notice that oapi.yaml was changed so much, reverted all the formatting related changes.